contrib/debian/netdata.postrm

```diff
@@ -16,45 +16,31 @@ dpkg-maintscript-helper dir_to_symlink \
 	/var/lib/netdata/www/static /usr/share/netdata/www/static 1.18.1~ netdata -- "$@"
 
 case "$1" in
-	remove)
-		;;
+remove) ;;
 
+\
 	purge)
```

Comment:
Shouldn't the indentation be similar to `remove)` and `*)` ?
```suggestion
purge)
```

Reply:
See above -- Same thing.

Reply:
```sh
$ shfmt --help
usage: shfmt [flags] [path ...]

If no arguments are given, standard input will be used. If a given path
is a directory, it will be recursively searched for shell files - both
by filename extension and by shebang.

  -version  show version and exit

  -l        list files whose formatting differs from shfmt's
  -w        write result to file instead of stdout
  -d        error with a diff when the formatting differs
  -s        simplify the code
  -mn       minify the code to reduce its size (implies -s)

Parser options:

  -ln str   language variant to parse (bash/posix/mksh, default "bash")
  -p        shorthand for -ln=posix

Printer options:

  -i uint   indent: 0 for tabs (default), >0 for number of spaces
  -bn       binary ops like && and | may start a line
  -ci       switch cases will be indented
  -sr       redirect operators will be followed by a space
  -kp       keep column alignment paddings

Utilities:

  -f        recursively find all shell files and print the paths
  -tojson   print syntax tree to stdout as a typed JSON
```

Reply:
I _think_ I've fixed this behaviour. For users of Vim and [vim-autoformat](https://github.com/chiel92/vim-autoformat) put this in your `$HOME/.vimrc`:

```
" auto-format
au BufWrite * :Autoformat
let g:autoformat_autoindent = 0
let g:autoformat_retab = 0
let g:autoformat_remove_trailing_spaces = 0

let g:formatdef_custom_shfmt = '"shfmt -i 2 -ci -sr"'
let g:formatters_sh = ['custom_shfmt']
```